components/brave_wallet/common/fil_address.cc

```diff
@@ -203,8 +242,17 @@ std::string FilAddress::EncodeAsString() const {
   // and https://github.com/multiformats/multibase/blob/master/multibase.csv
   std::string encoded_output = base::ToLowerASCII(
       base32::Base32Encode(input, base32::Base32EncodePolicy::OMIT_PADDING));
-  return network_ + std::to_string(static_cast<int>(protocol_)) +
-         encoded_output;
+  if (protocol_ == mojom::FilecoinAddressProtocol::DELEGATED) {
+    auto r = network_ + std::to_string(static_cast<int>(protocol_)) +
+             // Agent id
+             "10" +
+             // Delimiter between agent_id and pubkey
+             "f" + encoded_output;
+    return r;
```

Comment:
Just `network_ + "410f" + encoded_output` - function comment already explains what it is